_posts/2016-08-28-elections.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Announcing the first VoxPupuli Elections
+date: 2016-08-28
+github_username: roidelapluie
+twitter_username: roidelapluie
+---
+
+Vox Pupuli is about to live its first elections. This blogpost will give you the
+context and how to become a candidate.
+
+> Vox Pupuli is a collective of Puppet module, tooling and documentation authors
+> all working together to ensure continued development on the code we maintain.
+
+(Source: [https://voxpupuli.org/](https://voxpupuli.org/))
+
+Vox Pupuli now represents 68 collaborators, and 86 repositories. In order to
+make this project successful in the long term, we have been [discussing][d]
+about how we can better work together and take appropriate decisions.
+
+Open-Source Communities nowadays gain a lot from having a structure that can
+take action when bad things happens, and also when improvement is needed. We
+already have a [Code of Conduct][coc] in place, but we obviously need to ensure
+it is respected.
+
+On August 22, after abut 2 months of discussions, we merged the [Governance
+Document][gd]. It describes the responsibilities of the actors of Vox Pupuli:
+Users, Collaborators, Contributors… and it describes a Committee, the Project
+Management Committee (PMC).
+
+TL;DR The Committees members are elected by the community for 1 year. They take
+care of the Code of Conduct and its values, participate in strategic planning,
+amongst other things.
+
+In normal times, the PMC would elect an Election Officer to run the elections,
+as stated in the [Governance Document][gd]. However, to bootstrap the first
+elections, two members have volunteered to do that task:
+[Daniele "daenney" Sluijters][daenney] and [Julien "roidelapluie" Pivotto][me].
+I will take care of collecting the candidatures and promote the elections, while
+Daniele will run the actual elections.
+
+You can find more information [here][e], including how to become a candidate.
+Detailed information will be spread soon of this website too. Here is the
+timeline:
+
+* Before *September 17 2016 23:59UTC*: Accepting candidatures
+* Between *September 18 2016 12:00UTC* and *September 28 2016 12:00UTC*: Vote
+
+Please enjoy the party, and let's gather as much participation as possible
+during the whole election process!
```

Comment:
its there: "+Detailed information will be spread soon of this website too. Here is the
+timeline:"